src/cljs/main/broadfcui/page/library/library_page.cljs

```diff
@@ -22,76 +24,73 @@
-                                (if (= (:workspaceAccess data) "NO ACCESS") "You must request access to this dataset."))
-                     :content-renderer (fn [data]
-                                         (if (= (:workspaceAccess data) "NO ACCESS")
-                                           (icons/icon {:style {:alignSelf "center" :cursor "pointer"}
-                                                        :onClick #(react/call :check-access this data)} :shield)))}
-                    {:header-key "library:datasetName"
-                     :header (:title (:library:datasetName attributes)) :starting-width 250 :show-initial? true
-                     :as-text :library:datasetDescription :reorderable? false
-                     :content-renderer (fn [data]
-                                         (style/create-link {:text (:library:datasetName data)
-                                                             :onClick #(react/call :check-access this data)}))}
-                    {:header-key "library:indication"
-                     :header (:title (:library:indication attributes)) :starting-width 180 :show-initial? true}
-                    {:header-key "library:dataUseRestriction"
-                     :header (:title (:library:dataUseRestriction attributes)) :starting-width 180 :show-initial? true}
-                    {:header-key "library:numSubjects"
-                     :header (:title (:library:numSubjects attributes)) :starting-width 100 :show-initial? true}]
-                   (map
-                    (fn [keyname]
-                      {:header-key keyname
-                       :header (:title ((keyword keyname) attributes)) :starting-width 180 :show-initial? false})
-                    extra-columns))
-         :pagination (react/call :pagination this)
-         :->row (fn [data]
-                  (->> extra-columns
-                       (concat [:library:indication :library:dataUseRestriction :library:numSubjects])
-                       (map data)
-                       (cons data)
-                       (cons data)
-                       vec))}]))
+         {:items [[:div {:style {:fontSize "112%"}}
+                   [:span {:style {:fontWeight 700 :color (:text-light style/colors) :marginRight "0.5rem"}}
+                    "Matching Cohorts"]
+                   [:span {:style {:fontSize "80%"}}
+                    (let [total (or (:total @state) 0)]
+                      (str total
+                           " Dataset"
+                           (when-not (= 1 total) "s")
+                           " found"))]]
+                  flex/spring]
+          :style {:alignItems "flex-start" :marginBottom 7}
+          :column-edit-button {:style {:order 1 :marginRight nil}
+                               :anchor :right}}}]))
    :execute-search
    (fn [{:keys [refs]} reset-sort?]
      (let [query-params (merge {:current-page 1} (when reset-sort? {:sort-column nil :sort-order nil}))]
```

Comment:
`:current-page` doesn't seem to work. I think you meant `:page-number`.

To prevent this type of mistake, I think you can probably assert that `:update-query-params` may only get certain keys and anything else is an error. The logic in config.cljs may be helpful.